School placements page (#4213)
DFE-Digital/get-into-teaching-app: app/views/content/train-to-be-a-teacher/school-placements.md

    name: "Abbe, trainee teacher"
    classes: quote--indent
promo_content:
    - content/train-to-be-a-teacher/promos/adviser-promo-apply
keywords:
    - placements
    - school placements
    - classroom placements
    
---

School placements are an essential and exciting part of school-led or university-led postgraduate teacher training. They're a chance to:

* get experience in the classroom
* immerse yourself in your placement schools
* get to know your pupils and see their progress
* be part of a school community and learn from other teachers
* start your career as a teacher

$aaron$

## How long are placements?

School placements are designed to last for a minimum of 24 weeks. This time is spread across at least 2 schools. This is to give you a varied and thorough experience in the classroom. 

For example, in your first placement you might teach key stage 4, and in your second placement you might teach key stage 3 in a school for pupils with [special educational needs and disabilities (SEND)](/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs).

> You'll need experience of teaching in 2 schools to be awarded [qualified teacher status](/train-to-be-a-teacher/what-is-qts).

Author: ```suggestion
You'll need experience of teaching in 2 schools to be awarded [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts).
```

---
title: "Teacher training school placements"
heading: "Teacher training school placements"
subcategory: Postgraduate teacher training
description: |-
  Learn about teacher training school placements, including how long they are, what trainees do on placements, and the support available. 
navigation: 20.20
navigation_title: Teacher training school placements

> navigation_description: Learn about teacher training school placements, including how long they are, what trainees will do on placements, and the support available. 

Reviewer: ```suggestion
navigation_description: Learn about teacher training school placements, including how long they are, what trainees do on placements, and the support available. 
```

quote:
  q-giorgio:
    text: "The highlight of my placements was getting to teach!"
    name: "Giorgio, biology teacher"
    classes: quote--indent
promo_content:
    - content/train-to-be-a-teacher/promos/adviser-promo-apply
keywords:
    - placements
    - school placements
    - classroom placements
    
---

School placements are an important and exciting part of school-led or university-led postgraduate teacher training. They're a chance to:

* get experience in the classroom
* immerse yourself in your placement schools

> * get to know your pupils and colleagues

Author: ```suggestion
* get to know your pupils and see their progress
* be part of a school community and learn from other teachers
```

> * start your career as a teacher

Reviewer: Wonder if we could pull out the colleagues bit here, so become part of a school community and learn from other teachers, or something like that

## How long are placements?

School placements last for a minimum of 24 weeks. This time is spread across at least 2 schools. This is to give you a varied and thorough experience in the classroom. For example, in your first placement you might teach key stage 4, and in your second placement you might teach key stage 3 in a school for pupils with [special educational needs and disabilities (SEND)](/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs). 

## Where will my placements take place? 

> Your training provider will select placement schools for you. They will contact you and discuss your situation to help them select a location that you can travel to. 

Reviewer: Do they consider anything else? Eg if the trainee has any particular needs? Or is it just about travel?

Reviewer 2: Worth checking with policy

## What will I do on my placements?  

You school placement aims to give you enough experience to: 

* meet all the [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards)

* achieve [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) 

> On your school placement, you'll get hands-on experience of what it's like to be a teacher. So, you’ll be spending a lot of time in the classroom learning how to teach lessons. Beyond the classroom, you can also get involved in other aspects of school life, like school trips, school plays, parents' evenings and staff meetings.  

Reviewer: ```suggestion
On your school placement, you'll get hands-on experience of what it's like to be a teacher. So, you’ll be spending a lot of time in the classroom learning how to teach lessons. Beyond the classroom, you can get involved in other aspects of school life, like school trips, school plays, parents' evenings and staff meetings.  
```

> Over your placement you’ll get to see your pupils progress and get an insight into how rewarding teaching can be.  

Reviewer: ```suggestion
During your placement you’ll get to see your pupils progress and get an insight into how rewarding teaching can be.  
```

$q-giorgio$

## What support will I get during my placements? 

School placements can be daunting, but there's lots of support available.

### Your skills from training

> You’ll start your placements well equipped with a toolkit of theory and techniques that you'll learn in training. Placements are a chance to put that into practice.  

Reviewer: Should this be 'that you've learnt in training'?

Author: I don't think so, because this page should be targeting people who haven't started or applied to training yet? Unless we do 'you will have learnt'?

### Mentors

In each placement school, you'll have a dedicated mentor. Mentors are expert professionals and role models, who will give you knowledge, support and feedback throughout your placement to help you learn. 

Mentors also receive training on how to develop teachers and what trainees need to know to complete their course. 

### Tutors

> While your mentor is specific to each placement school, your tutor is consistant support from your training provider throughout your placements. They'll work with you and your mentors to make sure you have enough support and are making good progress towards your placement goals.  

Reviewer: ```suggestion
While your mentor is specific to each placement school, your tutor provides consistent support from your training provider throughout your placements. They'll work with you and your mentors to make sure you have enough support and are making good progress towards your placement goals.  
```

### School leadership and policies
Like qualified teachers, your school leadership team and colleagues will be there to support you. Your placement school may also have documents and policies in place to support their teachers. For example, all schools will have a behaviour policy that explains what's expected of pupils and what will happen if they misbehave.  

## Are placements paid? 

> For fee-funded postgraduate training, you can get financial support through [student finance](/funding-and-support/tuition-fee-and-maintenance-loans), or you may be eligible for [bursaries or scholarships](/funding-and-support/scholarships-and-bursaries). This support is available for your whole training course, including school placements. 

Reviewer: ```suggestion
For fee-funded postgraduate training, you can [get financial support through student finance](/funding-and-support/tuition-fee-and-maintenance-loans), or [you may be eligible for bursaries or scholarships](/funding-and-support/scholarships-and-bursaries). This support is available for your whole training course, including school placements. 
```

navigation: 20.20
navigation_title: Teacher training school placements
navigation_description: Learn about teacher training school placements, including how long they are, what trainees do on placements, and the support available. 
quote:
  jennifer:
    text: "My mentor was highly knowledgeable and experienced and had high expectations which helped me to flourish."
    name: "Jennifer, trainee teacher"
    classes: quote--indent
  aaron:
    text: "My first placement was an academy with an experienced teacher in a year 3 classroom. She took me under her wing and taught me all the basics on how to conduct myself in the classroom along with how to best motivate and nurture the children to promote positive behaviour responses and enhance the classroom culture."
    name: "Aaron, trainee teacher"
    classes: quote--indent
  abbe:
    text: "My mentor was the special educational needs co-ordinator. This meant I was able to learn more about this role and improve my understanding of the strategies and support available for children with special educational needs and disabilities (SEND)."

>     name: "Abbe, trainee teacher"

Author: I checked this as well - it's Abbe :)

    classes: quote--indent
promo_content:
    - content/train-to-be-a-teacher/promos/adviser-promo-apply
keywords:
    - placements
    - school placements
    - classroom placements
    
---

School placements are an important and exciting part of school-led or university-led postgraduate teacher training. They're a chance to:

* get experience in the classroom
* immerse yourself in your placement schools
* get to know your pupils and see their progress
* be part of a school community and learn from other teachers
* start your career as a teacher

$aaron$

## How long are placements?

School placements last for a minimum of 24 weeks. This time is spread across at least 2 schools. This is to give you a varied and thorough experience in the classroom. For example, in your first placement you might teach key stage 4, and in your second placement you might teach key stage 3 in a school for pupils with [special educational needs and disabilities (SEND)](/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs). 

## Where will my placements take place? 

> Your training provider will select placement schools for you. They will contact you and discuss your situation to help them select a location that you can travel to. 

Reviewer 2: ```suggestion
Your training provider will select placement schools for you. They will contact you and discuss your situation to help them select a location that you can travel to.
```

> ## What will I do on my placements?  

Reviewer 2: ```suggestion
## What will I do on my placements?
```

You school placement aims to give you enough experience to: 

* meet all the [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards)

> * achieve [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) 

Reviewer 2: ```suggestion
* achieve [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts)
```

> On your school placement, you'll get hands-on experience of what it's like to be a teacher. So, you’ll be spending a lot of time in the classroom learning how to teach lessons. Beyond the classroom, you can get involved in other aspects of school life, like school trips, school plays, parents' evenings and staff meetings.  

Reviewer 2: ```suggestion
On your school placement, you'll get hands-on experience of what it's like to be a teacher. So, you’ll be spending a lot of time in the classroom learning how to teach lessons. Beyond the classroom, you can get involved in other aspects of school life, like school trips, school plays, parents' evenings and staff meetings.
```

> During your placement you’ll get to see your pupils progress and get an insight into how rewarding teaching can be.  

Reviewer 2: ```suggestion
During your placement you’ll get to see your pupils progress and get an insight into how rewarding teaching can be.
```

$abbe$

> ## What support will I get during my placements? 

Reviewer 2: ```suggestion
## What support will I get during my placements?
```

School placements can be daunting, but there's lots of support available.

### Your skills from training
You’ll start your placements well equipped with a toolkit of theory and techniques that you'll learn in training. Placements are a chance to put that into practice.  

### Mentors

> 
> In each placement school, you'll have a dedicated mentor. Mentors are expert professionals and role models, who will give you knowledge, support and feedback throughout your placement to help you learn. 

Reviewer 2: ```suggestion

In each placement school, you'll have a dedicated mentor. Mentors are expert professionals and role models, who will give you knowledge, support and feedback throughout your placement to help you learn.
```

> Mentors also receive training on how to develop teachers and what trainees need to know to complete their course. 

Reviewer 2: ```suggestion
Mentors also receive training on how to develop teachers and what trainees need to know to complete their course.
```

$jennifer$

### Tutors

> While your mentor is specific to each placement school, your tutor provides consistent support from your training provider throughout your placements. They'll work with you and your mentors to make sure you have enough support and are making good progress towards your placement goals.  

Reviewer 2: ```suggestion
While your mentor is specific to each placement school, your tutor provides consistent support from your training provider throughout your placements. They'll work with you and your mentors to make sure you have enough support and are making good progress towards your placement goals.
```

### School leadership and policies

> Like qualified teachers, your school leadership team and colleagues will be there to support you. Your placement school may also have documents and policies in place to support their teachers. For example, all schools will have a behaviour policy that explains what's expected of pupils and what will happen if they misbehave.  

Reviewer 2: ```suggestion
Like qualified teachers, your school leadership team and colleagues will be there to support you. Your placement school may also have documents and policies in place to support their teachers. For example, all schools will have a behaviour policy that explains what's expected of pupils and what will happen if they misbehave.
```

> ## Are placements paid? 

Reviewer 2: ```suggestion
## Are placements paid?
```

> For fee-funded postgraduate training, you can [get financial support through student finance](/funding-and-support/tuition-fee-and-maintenance-loans), or [you may be eligible for bursaries or scholarships](/funding-and-support/scholarships-and-bursaries). This support is available for your whole training course, including school placements. 

Reviewer 2: ```suggestion
For fee-funded postgraduate training, you can [get financial support through student finance](/funding-and-support/tuition-fee-and-maintenance-loans), or [you may be eligible for bursaries or scholarships](/funding-and-support/scholarships-and-bursaries). This support is available for your whole training course, including school placements.
```

> For salaried teacher training, you’ll receive a salary throughout your teacher training, including placements. Because of this, your placement school may also be referred to as your ‘employing school’.  

Reviewer 2: ```suggestion
For salaried teacher training, you’ll receive a salary throughout your teacher training, including placements. Because of this, your placement school may also be referred to as your ‘employing school’.
```

promo_content:
    - content/train-to-be-a-teacher/promos/adviser-promo-apply
keywords:
    - placements
    - school placements
    - classroom placements
    
---

School placements are an important and exciting part of school-led or university-led postgraduate teacher training. They're a chance to:

* get experience in the classroom
* immerse yourself in your placement schools
* get to know your pupils and see their progress
* be part of a school community and learn from other teachers
* start your career as a teacher

$aaron$

## How long are placements?

School placements last for a minimum of 24 weeks. This time is spread across at least 2 schools. This is to give you a varied and thorough experience in the classroom. 

For example, in your first placement you might teach key stage 4, and in your second placement you might teach key stage 3 in a school for pupils with [special educational needs and disabilities (SEND)](/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs).

You'll need experience of teaching in 2 schools to be awarded [qualified teacher status](/train-to-be-a-teacher/what-is-qts).

## Where will my placements take place?

> Where you complete your school placements will depend on whether you've chosen a primary or secondary teacher training course.

Reviewer 2: This sounds slightly odd @emmaachesongray - it is not just dependent on primary vs secondary (although that is obviously a factor!). However, I appreciate this is what is in prod at the moment!

> Where you complete your classroom placements will depend on whether you've chosen a primary or secondary teacher training course.

Author: ```suggestion
Where you complete your school placements will depend on whether you've chosen a primary or secondary teacher training course.
```

> Your training provider will usually select placement schools for you. They'll contact you and discuss your situation to help them select the most appropriate locations. They'll take into consideration your course, subject, commute and personal circumstances.

Author: ```suggestion
Your training provider will usually select placement schools for you. They'll contact you and discuss your situation to help them select the most appropriate locations. They'll often take into consideration your course, subject, commute and other personal circumstances.
```

> Your training provider will select placement schools for you. They will contact you and discuss your situation to help them select a location that you can travel to.

Author: ```suggestion
Your training provider will usually select placement schools for you. They will contact you and discuss your situation to help them select a location that you can travel to.
```

> You school placements aims to give you enough experience to: 

Reviewer 2: ```suggestion
Your school placements aim to give you enough experience to: 
```

> * meet all the [Teachers’ Standards](https://www.gov.uk/government/publications/teachers-standards)

Reviewer 2: ```suggestion
* meet all the [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards)
```
our style (and GOV.UK's style!) is lower case :-) 
https://www.gov.uk/guidance/style-guide/a-to-z

You’ll start your placements well equipped with a toolkit of theory and techniques that you'll learn in training. Placements are a chance to put your new skills into practice.

### Mentors

In each placement school, you'll have a dedicated mentor. Mentors are expert professionals and role models, who will give you knowledge, support and feedback throughout your placement to help you learn.

Mentors also receive training on how to develop teachers and what trainees need to know to complete their course.

$jennifer$

### Tutors

While your mentor is specific to each placement school, your tutor provides consistent support from your training provider throughout your placements. They'll work with you and your mentors to make sure you have enough support and are making good progress towards your placement goals.

### School leadership and policies

Your school leadership team and colleagues will be there to support you. Your placement schools may also have documents and policies in place to support their teachers. For example, all schools will have a behaviour policy that explains what's expected of pupils and what will happen if they misbehave.

## Are placements paid?

> For fee-funded postgraduate training, you can [get financial support through student finance](/funding-and-support/tuition-fee-and-maintenance-loans), or [you may be eligible for bursaries or scholarships](/funding-and-support/scholarships-and-bursaries). This support is available for your whole training course, including school placements.

Reviewer 2: Just flagging that we have two links pretty close together here - whilst this might technically be OK re accessibility, is it worth considering whether this is another way to do this @emmaachesongray ?
